frontend/src/scenes/sceneLogic.ts

```diff
@@ -249,8 +251,8 @@ export const sceneLogic = kea<sceneLogicType>({
         return mapping
     },
     listeners: ({ values, actions }) => ({
-        showUpgradeModal: ({ featureName }) => {
-            posthog.capture('upgrade modal shown', { featureName })
+        showUpgradeModal: ({ featureNameBenefit }) => {
+            posthog.capture('upgrade modal shown', { featureNameBenefit })
```

Comment:
Even though this event reporting is quite simple, I would still move it to eventUsageLogic so we keep all that information centralized. I would definitely send the featureName too, actually might be more useful than the benefit, and it ensure backwards-compatibility too.